samples/mpsl/timeslot/sample.yaml

```diff
@@ -5,5 +5,5 @@ tests:
   samples.mpsl.timeslot:
     build_only: true
     build_on_all: true
-    platform_allow: nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf5340pdk_nrf5340_cpunet
+    platform_allow: nrf52dk_nrf52832 nrf52840dk_nrf52840
```

Comment:
MPSL timeslot should have nrf53 support. nrf5340dk_nrf5340_cpunet in this case

Reply:
done